package.json

```diff
@@ -174,8 +174,8 @@
 		"ajv": "8.7.1",
 		"ajv-draft-04": "1.0.0",
 		"appium": "2.0.0",
-		"babel-jest": "29.6.2",
-		"babel-loader": "8.2.3",
+		"babel-jest": "29.7.0",
+		"babel-loader": "9.1.3",
```

Comment:
[I think this is a major release because of changes to Node.js and webpack version support.](https://github.com/babel/babel-loader/releases/tag/v9.0.0) We shouldn't be affected.

> Require babel ^7.12.0 and Node.js >= 14.15.0 versions by @JLHwung in https://github.com/babel/babel-loader/pull/956
> Remove dependency on loader-utils and drop webpack 4 support by @nied in https://github.com/babel/babel-loader/pull/942